Eggstravaganza/Assets/Scripts/GameNetwork.cs

```diff
@@ -84,4 +102,53 @@ void EndRound()
             m_GameState.Value = GameState.EndRound;
         }
     }
+
+    void SpawnCountdown()
+    {
+        if (!IsOwner)
+            return;
+        
+        // Spawns an egg at a random time from 1 to maxEggSpawnTime seconds
+        if (m_TimerIsRunning)
+        {
+            if (m_TimeRemaining > 0)
```

Comment:
nit: could you combine `m_TimerIsRunning` with just checking `m_TimeRemaining > 0`?

Reply:
good point, done!